Release Notes/2015-06-11-cloud-platform-release-notes.md

```diff
@@ -71,7 +71,7 @@ Even though our global cloud platform is able to scale to various enterprise wor
 
 ### Open Source Contributions (2)
 
-* [__Lorry.io.__](//imagelayers.io/) Lorry.io is a CenturyLink Labs open-source project for creating, validating and sharing Docker Compose YAML files. Docker-compose.yml files are used to define and run complex applications with Docker and Docker Compose. Lorry.io can also import Panamax templates and convert them into docker-compose.yml files. For more information, [read this blog post](//www.centurylinklabs.com/lorry-io-pathway-to-docker-composable-apps/) and [browse the wiki here](//github.com/CenturyLinkLabs/lorry).
+* [__Lorry.io.__](https://lorry.io/) Lorry.io is a CenturyLink Labs open-source project for creating, validating and sharing Docker Compose YAML files. Docker-compose.yml files are used to define and run complex applications with Docker and Docker Compose. Lorry.io can also import Panamax templates and convert them into docker-compose.yml files. For more information, [read this blog post](/developers/blog/post/lorry-io-pathway-to-docker-composable-apps) and [browse the wiki here](//github.com/CenturyLinkLabs/lorry).
```

Comment:
Change link to `//www.ctl.io/developers/blog/post/lorry-io-pathway-to-docker-composable-apps`